test/integration/test_navbar_links.js

```diff
@@ -0,0 +1,139 @@
+tap.test('checkCreateLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "create")]/a';
+    var checkCreateLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkCreateLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/projects/editor/?tip_bar=home';
+        //the create href should match `/projects/editor/?tip_bar=home`
+        //the create href should be at the end of the URL
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkExploreLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "explore")]/a';
+    var checkExploreLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkExploreLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/explore/projects/all';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkDiscussLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "discuss")]/a';
+    var checkDiscussLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkDiscussLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/discuss';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkAboutLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "about")]/a';
+    var checkAboutLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkAboutLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/about';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+tap.test('checkHelpLinkWhenSignedOut', function (t) {
+    var xPathLink = '//li[contains(@class, "link") and contains(@class, "help")]/a';
+    var checkHelpLinkWhenSignedOut = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
+    checkHelpLinkWhenSignedOut.getAttribute('href').then( function (url) {
+        //expected value of the href
+        var expectedHref = '/help';
+        t.equal(url.substr(-expectedHref.length), expectedHref);
+        t.end();
+    });
+});
+
+// ==== Search bar ====
+
+tap.test('checkSearchBar', function (t) {
+    var xPathLink = '//input[@id="frc-q-1088"]';
+    // search bar should exist
+    // if it isn't found, it will throw an error, which tap will treat as a failure
+    var checkSearchBar = driver.findElement(seleniumWebdriver.By.xpath(xPathLink));
```

Comment:
I added an assertion, not sure if it was the best one to choose, but... I used ok() for now. I wanted to get this in a place where it made sense for @paulkaplan & I to talk about it together after our conversation earlier today, so I made a first-pass at addressing the issues in it (there's still 1 failing test, I haven't looked at why it's failing yet).